outlines/deployment.md

```diff
@@ -0,0 +1,69 @@
+# Deployment, monitoring and analytics
+
+1. Deploying a web application
+  1. A web application is not necessarily like any other piece of released software
+     1. Can be updated frequently
+     2. Still QA is important!
+  2. Deployment environments: production vs staging vs development
+    1. Staging should be as close as possible to production
+  3. Meteor ENV + settings
+    1. Use ENV for "host specific" things -- usually just MONGO_URLs and MAIL_URLs
+    2. Use settings for environment specific things, like 3rd party secrets.
+    3. Settings *should not* be part of the repository (too much trust in 3rd parties). Ideally stored in some secure location.
+2. Other things to consider when deploying
+  1. Domain names
+  2. SSL certificates
+  3. CDNs
+    1. What they are for
+    2. How they are usually used -- e.g. CloudFront (find a useful general article to link to)
+3. Deployment options
+  1. Deploying to .meteor.com via `meteor deploy`.
+    1. How to do it
+    2. Managing a deployment / deleting
+    3. Accessing mongo
+    4. Performance characteristics => don't use for production
+  2. Deploying with MUP
+    1. Description of what MUP does
+    2. Link to some articles w/ example content
+  3. Deploying with Modulus
+    1. Description of what Modulus is
+    2. Link to some articles w/ example content
+  4. Deploying with Galaxy
+    1. Description of Galaxy
+    2. How to do it
+    3. Management on the commandline
+    4. The Galaxy UI -- link to galaxy docs
+    5. How rolling app updates work, and why they are important
```

Comment:
Right. I do mention it in 4.4. I guess I just want to mention it as a selling point of Galaxy. I think this might be a section that others might want to weigh in on anyway, maybe I should keep it vanilla for now..